### PR TITLE
wpaclean: add page

### PR DESCRIPTION
**Pull request by johandegn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 1.7.0**

### CODE REVIEW

File: pages/common/wpaclean.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# wpaclean
+
+> Clean capture files to get only the 4-way handshake and a beacon.
+> Part of Aircrack-ng network software suite.
+> More information: <https://manned.org/wpaclean.1>.
+
+- Clean capture and save only 4-way handshake and a beacon in result:
```
Comment (kbdharun) [inline pages/common/wpaclean.md:7]:
```suggestion
- Clean capture and save only a 4-way handshake and a beacon in the result:
```

File: pages/common/wpaclean.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+`wpaclean {{path/to/result.cap}} {{path/to/capture.cap}}`
+
+- Clean multiple captures and save 4-way handshakes and beacons in result:
```
Comment (kbdharun) [inline pages/common/wpaclean.md:11]:
```suggestion
- Clean multiple captures and save 4-way handshakes and beacons in the result:
```

File: pages/common/wpaclean.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+`wpaclean {{path/to/result.cap}} {{path/to/capture1.cap path/to/capture1.cap ...}}`
```
Comment (kbdharun) [inline pages/common/wpaclean.md:13]:
```suggestion
`wpaclean {{path/to/result.cap}} {{path/to/capture1.cap path/to/capture2.cap ...}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**